libraries/Microsoft.Bot.Builder.AI.QnA/Utils/QnACardBuilder.cs

```diff
@@ -77,48 +77,82 @@ public static IMessageActivity GetSuggestionsCard(List<string> suggestionsList,
         }
 
         /// <summary>
-        /// Get active learning suggestions card.
+        /// Get Card for MultiTurn scenario. (Can be deprected from 4.10.0 release of sdk).
```

Comment:
Please copyedit and improve quality of documentation.

Reply:
Updated.

---
In reply to: [430556823](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430556823) [](ancestors = 430556823)